src/main/java/thePackmaster/cards/dimensiongatepack/Inferno.java

```diff
@@ -1,43 +1,65 @@
 package thePackmaster.cards.dimensiongatepack;
 
+import basemod.BaseMod;
+import com.evacipated.cardcrawl.mod.stslib.fields.cards.AbstractCard.PersistFields;
 import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
 import com.megacrit.cardcrawl.cards.AbstractCard;
-import com.megacrit.cardcrawl.cards.DamageInfo;
 import com.megacrit.cardcrawl.characters.AbstractPlayer;
 import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
 import com.megacrit.cardcrawl.monsters.AbstractMonster;
-import thePackmaster.actions.dimensiongatepack.SelfDamageAction;
+import com.megacrit.cardcrawl.relics.ChemicalX;
+import com.megacrit.cardcrawl.ui.panels.EnergyPanel;
+import com.megacrit.cardcrawl.vfx.combat.ScreenOnFireEffect;
+import thePackmaster.util.cardvars.HoardField;
 
 import static thePackmaster.SpireAnniversary5Mod.makeID;
 
 public class Inferno extends AbstractDimensionalCard {
     public final static String ID = makeID("Inferno");
-    // intellij stuff attack, enemy, basic, 6, 3, , , ,
 
     public Inferno() {
-        super(ID, 1, CardRarity.UNCOMMON, AbstractCard.CardType.ATTACK, AbstractCard.CardTarget.ALL);
-        baseDamage = 15;
-        exhaust = true;
+        super(ID, -1, CardRarity.RARE, AbstractCard.CardType.ATTACK, AbstractCard.CardTarget.ALL);
+        baseDamage = 100;
         setFrame("infernoframe.png");
         isMultiDamage = true;
-
+        PersistFields.basePersist.set(this, 2);
+        PersistFields.persist.set(this, 2);
+        HoardField.baseHoard.set(this, 9);
+        HoardField.hoard.set(this, 9);
```

Comment:
Interesting to see where this ended up... reasonably translating this effect from Monster Train seems hard.